app/knights_setup/knights_class.py

```diff
@@ -0,0 +1,15 @@
+from app.knights_setup.knight_class import Knight
+
+
+class Knights:

```

Comment:
Do not create classes with names in plural, you should avoid it. This functionality can be implemented in the form of a class attribute in the `Knight` class, implement the `all_knights` class attribute there as dict, use the name as key and `self` as value (would represent class instance) in the `__init__` there. Remove this file after implementation.